lib/Files.js

```diff
@@ -731,6 +738,7 @@ class Files {
    * @param {object} options Options to generate presign URL
    * @param {string} options.blobName file path
    * @param {number} options.expiryInSeconds presign URL expiry duration
+   * @param {string} options.permissions premissions for presigned URL
```

Comment:
could you add a validateInput for the options like in https://github.com/adobe/aio-lib-files/blob/784b6fffe07b2db369702983664020a9443d8153/lib/Files.js#L430 for example ?